pyo3-ffi/src/pyframe.rs

```diff
@@ -1,16 +1,16 @@
+#[allow(unused_imports)]
+use crate::object::PyObject;
 #[cfg(not(GraalPy))]
 #[cfg(any(Py_3_10, all(Py_3_9, not(Py_LIMITED_API))))]
 use crate::PyCodeObject;
-#[cfg(not(Py_LIMITED_API))]
 use crate::PyFrameObject;
-use std::os::raw::c_int;
-
-#[cfg(Py_LIMITED_API)]
-opaque_struct!(pub PyFrameObject);
+#[allow(unused_imports)]
+use std::os::raw::{c_char, c_int};
```

Comment:
`c_char` doesn't seem to be used 🤔 

```suggestion
use std::os::raw::c_int;
```